Cargo.toml

```diff
@@ -5,5 +5,6 @@ members = [
     "light-node",
     "rpc",
     "tendermint",
+    "tendermint-proto",
```

Comment:
Yup, I thought that might come up. Would you be fine calling it `proto`? That's what it is (going to be) in the ibc-rs repo.

Reply:
I renamed the folder to `proto` and moved the compiler into the root `proto-compiler` folder. Since the compiler requires weird dependencies, like `git clone` for builds, I excluded it from the workspace.